server/server.js

```diff
@@ -2,11 +2,14 @@
 const express = require('express');
 const path = require('path');
 const routes = require('./routes');
+const bodyParser = require('body-parser')
 
 const app = express();
 
 //TODO We need middleware to convert request and response payloads to an from JSON
+app.use(bodyParser.json())
```

Comment:
@eliascodes this should be fixed now. I could restructure the solution folder to have just a copy of the whole server folder with the files that are different changed, but I feel like it's probably this sort of thing that will lead to the same problem that this PR is trying to fix (things being different in files the students aren't expected to change in a solution)